prisma/migrations/20241124130447_made_links_shorturl_unique/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[shortUrl]` on the table `Link` will be added. If there are existing duplicate values, this will fail.
```

Comment:
**issue**: Ensure there are no existing duplicate `shortUrl` values in the `Link` table before applying this migration to prevent failure.